domain/src/test/java/camp/nextstep/edu/calculator/CalculatorTest.kt

```diff
@@ -0,0 +1,40 @@
+package camp.nextstep.edu.calculator
+
+import com.camp.nextstep.edu.domain.Calculator
+import com.google.common.truth.Truth.assertThat
+import org.junit.Test
+
+class CalculatorTest {
+    @Test
+    fun `1 + 2 + 3`() {
+        val calculator = Calculator()
+        val actual = calculator.evaluate("1 + 2 + 3")
+
+        assertThat(actual).isEqualTo(6)
+    }
+
+    @Test
+    fun `곱셉 나눗셈 덧셈 뺄샘`() {
+        val calculator = Calculator()
+        val actual = calculator.evaluate("1 ++ 2 / 3 * 2 - 3")
+
+        assertThat(actual).isEqualTo(-1)
+    }
+
+    @Test
+    fun `1+ 2 + 3 +`() {
+        val calculator = Calculator()
+        val actual = calculator.evaluate("1+ 2 + 3 +")
+
+        assertThat(actual).isEqualTo(6)
+    }
+
+    @Test
+    fun `+ 1+ 2 + 3`() {
```

Comment:
위의 테스트코드와 동일하게 에러가 발생해야하는 테스트가 아닐까요?

Reply:
잘못된 수식어인 경우를 테스트하기 위함이였습니다.
혹시 적절한 테스트 코드방법이 있을까요?

Reply:
이제 테스트가 잘 통과하네요! 
테스트케이스명을 변경해주면 어떨까요?
```suggestion
    fun `+ 1+ 2 + 3 계산식이 주어졌을때, 계산하면, 에러가 발생한다`() {
```
잘짜여진 테스트 케이스들은 하나의 요구사항 명세서로 보기도합니다 😄